File: java-home-inventory-core/src/br/com/sailboat/homeinventory/core/entity/Shopping.java
package br.com.sailboat.homeinventory.core.entity;

import java.util.Calendar;

public class Shopping {

    private Long id;
    private Establishment establishment;
    private Calendar purchaseDateTime;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Establishment getEstablishment() {
        return establishment;
    }

    public void setEstablishment(Establishment establishment) {
        this.establishment = establishment;
    }

    public Calendar getPurchaseDateTime() {
        return purchaseDateTime;
    }

    public void setPurchaseDateTime(Calendar purchaseDateTime) {
        this.purchaseDateTime = purchaseDateTime;
    }
}

File: android-home-inventory/app/src/main/java/br/com/sailboat/homeinventory/model/Shopping.kt
package br.com.sailboat.homeinventory.model

import br.com.sailboat.canoe.helper.EntityHelper
import java.util.*

data class Shopping(
    var id: Long = EntityHelper.NO_ID,
    var establishmentId: Long = EntityHelper.NO_ID,
    val dateTime: Calendar
)